integration_tests/test_suites/k8s-test-suite/tests/test_utils.py

```diff
@@ -320,6 +320,36 @@ def test_pod_debug_info_failure(cluster_provider, namespace, should_cleanup):
         assert " whoops!\n" in pod_debug_info
         assert pod_debug_info.endswith("No warning events for pod.")
 
+        # Test case where the pod unexpectedly terminates and logs collection is skipped
+        api_client.batch_api.create_namespaced_job(
+            body=construct_job_manifest("failpoddebugnologs", 'echo "whoopsies!"; exit 1'),
+            namespace=namespace,
+        )
+
+        with pytest.raises(
+            DagsterK8sError,
+            match="Encountered failed job pods for job failpoddebugnologs with status:",
+        ):
+            api_client.wait_for_job_success("failpoddebugnologs", namespace=namespace)
+
+        pod_names = api_client.get_pod_names_in_job("failpoddebugnologs", namespace=namespace)
```

Comment:
do you need to set up a separate k8s job for this? Or can you use the same job from the test case just above, with the skip_logs argument changed to True instead of False, and verify that the resulting debug info is different?